src/css/campaignPage.css

```diff
@@ -94,23 +93,29 @@ body {
     font-weight: 100;
 }
 
+.mb-3.input-group{
```

Comment:
where is this class used?

Reply:
this is the numerical input for donations inside the campaign detail.